goyaml.v3/encode_test.go

```diff
@@ -19,6 +19,7 @@ import (
 	"bytes"
 	"fmt"
 	"math"
+	yaml "sigs.k8s.io/yaml/goyaml.v3"
```

Comment:
nit: spurious import reordering? this doesn't seem like it belongs with stdlib imports

Reply:
fixed